# File: waywiser_perception/msg/Detection.msg

int32 class_id
float64 confidence
waywiser_perception/BoundingBox bbox_2d
waywiser_perception/BoundingBox bbox_3d
